content/introduction.md

```diff
@@ -45,3 +45,8 @@ You can also manually create a new project from scratch by installing the core a
 ```bash
 $ npm i --save @nestjs/core @nestjs/common rxjs reflect-metadata
 ```
+
+#### TC39 decorators proposal
+
+Please if you enjoy nestjs, go and star repo with the [proposal-decorators
```

Comment:
can you remove the break line?

![image](https://user-images.githubusercontent.com/13461315/147794174-c3437bfe-dd89-42a1-86b2-4b556f26f42b.png)